clang/lib/CodeGen/CGCall.cpp

```diff
@@ -5692,7 +5692,7 @@ RValue CodeGenFunction::EmitCall(const CGFunctionInfo &CallInfo,
         !getLangOpts().FPAccuracyVal.empty()) {
       const auto *FD = dyn_cast_if_present<FunctionDecl>(TargetDecl);
       if (FD) {
-        CI = EmitFPBuiltinIndirectCall(IRFuncTy, IRCallArgs, CalleePtr, FD);
+        CI = EmitFPBuiltinofFD(IRFuncTy, IRCallArgs, CalleePtr, FD);
```

Comment:
Does it make sense to add a check here to see if this is a function that needs fpbuiltin support?

Reply:
I am checking this in the function EmitFPBuiltinofFD. If the function FD has not BuiltinID  but is one of the function listed in EmitFPBuiltinofFD (~ line# 22861), an attribute is generated for it.  Is that what you mean, or am I missing something?

Reply:
I just thought that if you checked it here you wouldn't need to call EmitPDBuiltinofFD at all in many cases.